packages/app/package.json

```diff
@@ -74,10 +74,10 @@
   },
   "sdkVersions": {
     "ios": {
-      "firebase": "11.15.0",
-      "iosTarget": "13.0",
+      "firebase": "12.0.0",
+      "iosTarget": "15.0",
```

Comment:
should be merged with something like 

```
BREAKING CHANGE: iOS mininum target now 15.0, was 13.0
```

Should not be too big a deal as current react-native already bumped theirs (we were running 15.1 IPHONEOS_DEPLOYMENT_TARGET in our e2e test anyway?), we supported a wider range than they do I think and are now just trimming old versions to catch up

Also, developer environment minimums were bumped:

https://github.com/invertase/react-native-firebase/blob/fe4550f362548dcb66359044715e22222a413f97/docs/index.md?plain=1#L21

It is now Xcode 16.2 as a minimum per https://firebase.google.com/support/release-notes/ios#version_1200_-_july_15_2025 which implies a minimum macOS of 14.5 (macOS Sequoia) per https://en.wikipedia.org/wiki/Xcode#Xcode_15.0_-_16.x_(since_visionOS_support)